### ISSUE TITLE
loss become nan

### ISSUE THREAD

**Issue by jiazhenggod** [titu1994/neural-image-assessment#40]
hello! Just got another question.When i train the nasnet mobile model, the loss becomes nan during the fifth epoch. Any possible reason for that? Should i tune some parameters? I used some other data before and it did not happen. Thanks

**Comment by titu1994** [comment]
It may be because of many things. Try running the learning rate.
